stock_reception_screen/__manifest__.py

```diff
@@ -27,6 +27,6 @@
         "views/stock_reception_screen.xml",
         "views/manual_barcode.xml",
     ],
-    "installable": True,
+    "installable": False,
```

Comment:
@dzungtran89 sorry my mistake... the module was already flagged `installable` in the `pre-commit` commit, I was reviewing only your migration commit and didn't see the change.

Reply:
@sebalix okay, I missed this comment too :(. Just pushed the PR